src/content/en/updates/2020/01/devtools.md

```diff
@@ -0,0 +1,153 @@
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 81) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Moto G4 support in Device Mode {: #motog4 }
+
+After [enabling the Device Toolbar](/web/tools/chrome-devtools/device-mode#viewport) you can now
+simulate the dimensions of a Moto G4 viewport from the **Device** list.
+
+![Simulating a Moto G4 viewport](/web/updates/images/2020/01/motog4.png)
+
+Click [Show Device Frame](/web/tools/chrome-devtools/device-mode/#frame) to show the Moto G4 hardware
+around the viewport.
+
+![Showing the Moto G4 hardware](/web/updates/images/2020/01/motog4frame.png)
+
+Related features:
+
+* Open the [Command Menu](/web/tools/chrome-devtools/command-menu/) and run the `Capture screenshot`
+  command to take a screenshot of the viewport that includes the Moto G4 hardware (after enabling
+  **Show Device Frame**).
+* [Throttle the network and CPU](http://localhost:8080/web/tools/chrome-devtools/device-mode/#throttle)
+  to more accurately simulate a mobile user's web browsing conditions.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/49e4673403214c99cc9d358fb5f311727dcf47e8 #}
+
+[Chromium issue #924693](https://crbug.com/924693)
+
+## Cookie-related updates {: #cookies }
+
+### Blocked cookies in the Cookies pane {: #blockedcookies }
+
+The Cookies pane in the Application panel now colors blocked cookies with a yellow background.
+
+![Blocked cookies in the Cookies pane of the Application panel](/web/updates/images/2020/01/blockedcookies.png)
+
+See also [Debug why a cookie was blocked](/web/updates/2019/10/devtools#blockedcookies) to learn
+how to access a similar UI from the Network panel.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/341eea56a810370d6f36ef6dd750df9cd66cd398 #}
+
+[Chromium issue #1030258](https://crbug.com/1030258)
+
+### Cookie priority in the Cookie pane {: #cookiepriority }
+
+The Cookies tables in the Network and Application panels now include a **Priority** column.
+
+<aside class="caution">
+  Chrome is the only browser that supports cookie priority.
+</aside>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e65507201ae7c4b83f297bab97e4f4dcc1e1168d #}
+
+[Chromium issue #1026879](https://crbug.com/1026879)
+
+### Edit all cookie values {: #edit }
+
+All cells in the Cookie tables are editable now, except cells in the **Size** column because that
+column represents the network size of the cookie, in bytes. 
+See [Fields](/web/tools/chrome-devtools/storage/cookies#fields) for an explanation of each column.
+
+![Editing a cookie value](/web/updates/images/2020/01/editcookie.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/cc93308cee322f28b02be92ac410d6651938de37 #}
+
+### Copy as Node.js fetch to include cookie data {: #fetchcookies }
```

Comment:
Being able to copy cookie data seemed to be the main value-add here hence why I lumped it in with the rest of the cookies content

Reply:
Lumping it in here SGTM.